static/app/views/insights/queues/settings.ts

```diff
@@ -2,6 +2,8 @@ import {t} from 'sentry/locale';
 
 export const MODULE_TITLE = t('Queues');
 export const BASE_URL = 'queues';
+export const DATA_TYPE = t('Queue');
```

Comment:
🤔 not `t('Task')`?

Reply:
Spoke with Ed, We're going to keep it as Queue